### DISCUSSION TITLE
How to use typescript-plugin-vue

### DISCUSSION THREAD

**Discussion by Mister-Hope** [vuedx/languagetools#234] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/33315834/121361599-0570ea80-c968-11eb-9af6-a7ef49779214.png)

I came to here because I just read the readme file generated by vite template.

I want to use this plugin, but the [readme](https://github.com/znck/vue-developer-experience/tree/main/packages/typescript-plugin-vue) **is not mentioning a single word** about how to config the plugin, but says a lot on what the plugin can do and the plugin architecture

I am setting this, but it do not work:

![image](https://user-images.githubusercontent.com/33315834/121361940-4bc64980-c968-11eb-8571-ee7a8eaf515a.png)

My biggest concren is what should I set to make this plugin work? Thanks. That's what a readme.md file should do.

**Comment by kevbot-git** [comment]
@znck could you please explain this? I have this issue also

**Comment by znck** [comment, marked as answer]
For Visual Studio Code, you just need to install [VueDX extension](https://marketplace.visualstudio.com/items?itemName=znck.vue-language-features). Just one caveat, use pre-release version.

![image](https://user-images.githubusercontent.com/2596484/162633469-249d747d-670d-42cc-868b-20a127eb56e4.png)

Added a [Getting started](https://github.com/vuedx/languagetools#getting-started) section to `readme.md`.

**Reply by blocka** [reply]
I have installed this extension, and the npm module, made sure I'm using workspace version of TS, and added the plugin to tsconfig. I'm still seeing "Cannot find module *.vue", etc.

**Reply by znck** [reply]
Is this project open source?
